xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -85,12 +89,22 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
     String newBalancerName = xdsConfig.balancerName;
     if (!newBalancerName.equals(balancerName)) {
       balancerName = newBalancerName; // cache the name and check next time for optimization
-      lookasideChannelLb.switchTo(newLookasideChannelLbProvider(newBalancerName));
+      Node node = resolvedAddresses.getAttributes().get(XDS_NODE);
+      if (node == null) {
+        node = Node.newBuilder()
```

Comment:
Use `Node.getDefaultInstance()`. We are assuming bootstrap never fails now. When we are handling the case bootstrap could fail, load balancing would never happen. So do not do thing more than that.

Reply:
Our current interop test is not bootstrap, and it's not using XdsResolver.

Reply:
For that purpose, work it around in `XdsComms2.

Reply:
The difference between current interop test and bootstrap is at this point, bootstrap sets attributes and interop does not. So it's better to handle it here.

Reply:
Ok, just realized the interop test is not using `XdsNameResolver`, so node is not in`resolvedAddresses` for sure.